internal/buildscripts/packaging/msi/splunk-otel-collector.wxs

```diff
@@ -36,6 +36,9 @@
                         Stop="both"
                         Remove="uninstall"
                         Wait="yes" />
+                     <RegistryKey Root="HKLM" Key="SYSTEM\CurrentControlSet\Services\EventLog\Application\splunk-otel-collector">
+                        <RegistryValue Type="expandable" Name="EventMessageFile" Value="C:\Windows\Microsoft.NET\Framework64\v4.0.30319\EventLogMessages.dll"/>
```

Comment:
How reliable is this path across supported OS versions?

Reply:
Good point. I'll change it to `%SystemRoot%\System32\EventCreate.exe` which is the same as the smart agent.